sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/Models/ClientSideEncryptionAlgorithm.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Storage.Cryptography.Models
+{
+    /// <summary>
+    /// Specifies the encryption algorithm used to encrypt and decrypt a blob.
+    /// </summary>
+    internal readonly struct ClientSideEncryptionAlgorithm
```

Comment:
should this be enum ?

Reply:
We can. It would be just as much code maintenance, though, because I'd have to write the custom serialize/deserialize logic. the enum name `AES_CBC_256` won't get through OneSDK code review (I believe they moved me to this).

Reply:
If this is internal only we may consider just using string constants then.